nextcord/role.py

```diff
@@ -46,6 +45,8 @@
         RoleTags as RoleTagPayload,
     )
     from .types.guild import RolePositionUpdate
+    from .message import Attachment
+    from .file import File
```

Comment:
I'm pretty sure the `File` imports need to be outside `if TYPE_CHECKING` if you are using it in `isinstance`.

If it works, you can ignore this, but I expect this would give a NameError since it's a runtime check.

Reply:
![image](https://user-images.githubusercontent.com/39167433/165475110-dd6f4c85-a1ed-46e2-907e-162af6b9a454.png)
 ## You were right.. Ima fix it